docs/index.md

```diff
@@ -0,0 +1,39 @@
+(_proxy_py_index)=
+# {{ project }} Documentation
+
+```{spelling}
+acceptor
+acceptors
+Changelog
+decrypt
+decrypted
+discoverable
+DNS
+Facebook
+http
+macOS
+Pre
+Py
+Scalable
+```
+
+```{include} ../README.md
+:end-before: (DO-NOT-REMOVE-docs-badges-START)
+```
+
+```{include} ../README.md
+:start-after: (DO-NOT-REMOVE-docs-badges-END)
+```
+
+```{toctree}
```

Comment:
@abhinavsingh you can have as many ToC trees as you want. Each may have a caption if needed. `:hidden:` is needed to exclude from the document render because this is also included in the sidebar and is unnecessary in the doc bottom.